plat/kvm/Linker.uk

```diff
@@ -1,8 +1,7 @@
 ifeq (x86_64,$(CONFIG_UK_ARCH))
-ifeq ($(CONFIG_KVM_BOOT_PROTO_MULTIBOOT),y)
+ifeq ($(findstring y,$(CONFIG_KVM_BOOT_PROTO_MULTIBOOT) $(CONFIG_KVM_BOOT_PROTO_MULTIBOOT2)),y)
 KVM_LDFLAGS-y += -Wl,-m,elf_x86_64
 KVM_LDFLAGS-y += -Wl,--entry=_multiboot_entry
-ELF64_TO_32 = y
```

Comment:
I think you misunderstood me. What I meant was having the following git commit history
- commit that removes this ELF64_TO_32 and replaces its if below with the MULTIBOOT config (mb2 support not in yet
- commit that introduces the script
- commit that adds multiboot2 support (C/asm changes and the addition to Linker.uk of multiboot2.py)
- commit that adds mb2 support and changes the function names in the mkukimg script.